src/dmd/dscope.d

```diff
@@ -273,6 +274,19 @@ struct Scope
     }
 
 
+    extern (C++) Scope* startRelaxShared()
+    {
+        Scope* sc = this.push();
+        sc.flags = this.flags | SCOPE.nosharedcheck;
```

Comment:
Why is the function named "RelaxShared" and the associated flag is "nosharedcheck" which bears no obvious nameing relationship?